server/model/data_stores.go

```diff
@@ -77,6 +79,8 @@ func DataStoreFromConfig(dsc config.TracingBackendDataStoreConfig) DataStore {
 		ds.Type = openapi.OPEN_SEARCH
 	case signalfx:
 		ds.Type = openapi.SIGNAL_FX
+	case newrelic:
+	case lighstep:
```

Comment:
If you have `newrelic` or `lighstep` in your config file, is this going to change the type to `otlp`? Do we want that behavior?

Reply:
Nice catch, fixed!